cmd/rekor-server/e2e_test.go

```diff
@@ -221,3 +221,77 @@ func getTreeID(t *testing.T) int64 {
 func TestSearchNoEntriesRC1(t *testing.T) {
 	util.RunCliErr(t, "search", "--email", "noone@internetz.com")
 }
+func TestGetNonExistentIndex(t *testing.T) {
+	// this index is extremely likely to not exist
+	out := util.RunCliErr(t, "get", "--log-index", "100000000")
+	util.OutputContains(t, out, "404")
+}
+func TestVerifyNonExistentIndex(t *testing.T) {
+	// this index is extremely likely to not exist
+	out := util.RunCliErr(t, "verify", "--log-index", "100000000")
+	util.OutputContains(t, out, "entry in log cannot be located")
+}
+
+func TestGetNonExistentUUID(t *testing.T) {
+	// this uuid is extremely likely to not exist
+	out := util.RunCliErr(t, "get", "--uuid", "ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff")
+	util.OutputContains(t, out, "404")
+}
```

Comment:
as these are CLI tests, should the be underneath `cmd/rekor-cli/...`?

Reply:
Yes, I have moved them. Thanks